projects/cdk/schematics/ng-update/v4/steps/constants/migration-warnings.ts

```diff
@@ -0,0 +1,58 @@
+import {MigrationWarning} from '../../../interfaces';
+
+export const MIGRATION_WARNINGS: MigrationWarning[] = [
+    {
+        name: 'tuiCreateTimeMask',
+        moduleSpecifier: '@taiga-ui/kit',
+        message:
+            'tuiCreateTimeMask has been removed. Use Time mask from Maskito instead https://github.com/taiga-family/maskito',
```

Comment:
Should we use more accurate url for `@maskito/kit` ? 🤔 

https://maskito.dev/kit/time

(the same for `tuiCreateDateRangeMask`, `tuiCreateDateMask` etc.)